### ISSUE TITLE
5 reliability anti-patterns

### ISSUE THREAD

**Issue by teivah** [sre-france/meetups#125]
### Author's Name

Teiva Harsanyi

### Author's Bio

SRE at Google, [author](https://www.amazon.com/100-Mistakes-How-Avoid-Them/dp/1617299596).

More: [teivah.io](http://teivah.io)

### Expected time

Standard talk (~40 min)

### Language

- [X] French
- [ ] English

### Abstract

Discussing common anti-patterns preventing your company from achieving an efficient level of reliability.
We will introduce reliability, then delve into common technical and organizational problems and how to prevent them.

Note: the number of anti-patterns may vary; it could be a bit more.

**Comment by pabluk** [comment]
Hi @teivah,

First of all, we really thank you for your participation in the Call for Proposal for SRE SummerCamp 2023.

We are very happy to announce that your talk has been approved for SRE SummerCamp 2023!

To confirm your participation, please just reply to this message **before 15 August 2023** and if you have any questions do not hesitate to contact us at srefranceasso+summercamp@gmail.com

Best,

SRE SummerCamp Team

**Comment by teivah** [comment]
@pabluk Hello, I'm sorry but I will have to cancel :/ There's a business trip that came in this week, it'll be too tight schedule-wise to make it to Chambéry.
Again sorry about that but I would be open, if possible, to give the presentation at another event (I live in Paris).

**Comment by pabluk** [comment]
Hi @teivah no worries, It would be great to have you at the SummerCamp but as you said maybe you could give the presentation in a future event, btw we're organizing a next Meetup in Paris on November.

**Comment by teivah** [comment]
@pabluk Is the date already known?

**Comment by pabluk** [comment]
> @pabluk Is the date already known?

Yes, it's not officially announced yet but it's scheduled on Wednesday, November 8. If you want I can sent you an email when all the details will be announced just to check your availability at that time.

**Comment by teivah** [comment]
Yes, that would be great :) Thanks

Le sam. 5 août 2023 à 10:25, Pablo Seminario ***@***.***> a
écrit :

**Comment by pabluk** [comment]
Hello @teivah,

I hope you're doing well, as mentioned in the previous comment, we have the date for next SRE meetup in Paris, it will take place on Wednesday, November 22, at Padok offices, so if you're available and want to join us as speaker we can allocate an slot for your presentation. Thanks

Cheers,

**Comment by teivah** [comment]
Hello @pabluk, cool! Count me in. How long should the talk be?

**Comment by pabluk** [comment]
> Hello @pabluk, cool! Count me in. How long should the talk be?

Great! regarding the duration, if it can last approximately 30/35 minutes it would be perfect.

So I'll include your presentation in the agenda of the event and I'll send you the link with all the details.

Thank you again @teivah for your contribution!

Best,

**Comment by pabluk** [comment]
Hi @teivah,

Here you can find all the details about the meetup https://www.meetup.com/site-reliability-engineering-france/events/296967843/ and I'll contact you some days before the event with more details about venue and if you need wifi, specific connectivity HDMI, etc.

In the meantime if you have any questions, please don't hesitate to comment here or contact us to srefranceasso@gmail.com. Thanks.

Cheers,
